assets/src/edit-story/components/panels/stylePreset/header.js

```diff
@@ -80,36 +107,35 @@ function PresetsHeader({
   const hasPresets = colors.length > 0;
 
   const getActions = () => {
-    return !isEditMode ? (
+    return (
```

Comment:
Same question here, was DRY the main reason for combining the two buttons into one, or also accessibility-related?

Reply:
Focus management again.